protostar/configuration_file/confiuration_toml_writer_test.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import dataclass
```

Comment:
I'm missing a test that verifies that `ConfigurationTOMLWriter` writes stuff in fixed order. So that our writes will not cause nondeterministic diffs.

Reply:
`test_saving_configuration` is a snapshot test, and it doesn't fail randomly